tutorials/spa-create-automation-api-trigger/spa-create-automation-api-trigger.md

```diff
@@ -101,24 +101,39 @@ Your **API Key** is now created and ready to be used.
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 3: ](Create service binding)]
-1.  Log in to your **SAP BTP Cockpit** and navigate to your Subaccount.
+[ACCORDION-BEGIN [Step 3: ](Create instance and service binding)]
+1.  Navigate to **SAP BTP Cockpit** subaccount > **Services** > **Instances and subscriptions**.
 
-2.  Create a new **Service Binding** by navigating to **Services** > **Instances and Subscriptions** > **Instances** > Click on your **SAP Process Automation Service** instance > Click **...** > **Create Service Binding**.
+2.  Choose **Create**.
 
-    !![Navigate to Service Bindings creation tab](Step3-1.png)
+    !![navigate to instances and subscriptions](Step3-1.png)
```

Comment:
Please correct typo.

Reply:
Hi Michael,

I'm not sure of understanding your comment, do you mean change it to capitals like this: "Navigate to Instances and Subscriptions" ?

Reply:
Just the N at the beginning.